src/e2e-test/java/io/cdap/plugin/bigquery/stepsdesign/BigQuerySink.java

```diff
@@ -45,6 +49,91 @@ public void sinkIsBigQueryMultiTable() {
     selectSinkPlugin("BigQueryMultiTable");
   }
 
+  @Then("Open the BiqQueryMultiTable sink properties")
```

Comment:
Why do we need these steps here? Is it not possible to re-use the steps from the framework?